src/oas/transformers/schema/keywords/format.ts

```diff
@@ -15,6 +15,7 @@ const ranges = {
 
 const createNumberFormatter = (min: number, max: number): Converter => {
   return schema => {
+    schema['x-stoplight'] = { explicitProperties: Object.keys(schema) };
```

Comment:
actually we should probably do
```suggestion
    (schema['x-stoplight'] ??= {}).explicitProperties = Object.keys(schema);
```
because an object may already have `x-stoplight` property

Reply:
oooo good point - I went ahead and changed it to `(schema['x-stoplight'] ??= {}).explicitProperties = Object.keys(schema).filter(word => word !== 'x-stoplight');` `x-stoplight` since `x-stoplight` will never be considered explicitly set